lib/backend/etcdv3/watcher.go

```diff
@@ -87,20 +87,35 @@ func (wc *watcher) watchLoop() {
 	// When this loop exits, make sure we terminate the watcher resources.
 	defer wc.terminateWatcher()
 
+	// If we are not watching a specific resource then this is a prefix watch.
```

Comment:
I think you copied this comment from below and also left it there.

Probably belongs in its original location rather than here?

Reply:
Thanks, fixed in bd21637f122daad89f15c2e049a123bee54e5db5